pridepy/files/files.py

```
@@ -669,14 +801,127 @@ def save_checksum_file(accession, output_folder):
+                    f"Downloading {file_record['fileName']} via {protocol} "
+                    f"(attempt {attempt}/{max_protocol_retries})"
+                )
+                try:
+                    Files._remove_if_exists(local_path)
+                    self._download_one_file_by_protocol(
+                        file_record,
+                        output_folder,
+                        protocol,
+                        aspera_maximum_bandwidth,
+                    )
+                except Exception as error:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)